### ISSUE TITLE
Cmake error if ACTS_BUILD_PLUGIN_ACTSVG=ON

### ISSUE THREAD

**Issue by DraTeots** [acts-project/acts#1818]
## Bug prerequisites

1. When ACTS v21.1.1 is built with flags: 

   ```bash
     cmake -Wno-dev -DCMAKE_INSTALL_PREFIX=/home/user/acts/acts-v21.1.1 
     -DCMAKE_CXX_STANDARD=17 
     -DACTS_BUILD_PLUGIN_TGEO=ON 
     -DACTS_BUILD_PLUGIN_DD4HEP=ON 
     -DACTS_BUILD_PLUGIN_JSON=ON 
     -DACTS_BUILD_PLUGIN_ACTSVG=ON  
     /home/user/acts/src/v21.1.1&& cmake --build . -- -j 4&& cmake --build . --target install
   ```
   
2. Cmake message confirms installation of Actsvg plugin: 

    ```
    ...
    -- Enable component 'Core' in 'Core'
    -- Ignore component 'PluginAutodiff' in 'Plugins/Autodiff'
    -- Enable component 'PluginActSVG' in 'Plugins/ActSVG'
    -- Ignore component 'PluginCuda' in 'Plugins/Cuda'
    -- Enable component 'PluginIdentification' in 'Plugins/Identification'
    -- Enable component 'PluginJson' in 'Plugins/Json'
    -- Ignore component 'PluginLegacy' in 'Plugins/Legacy'
    -- Ignore component 'PluginOnnx' in 'Plugins/Onnx'
    -- Ignore component 'PluginSycl' in 'Plugins/Sycl'
    -- Ignore component 'PluginExaTrkX' in 'Plugins/ExaTrkX'
    -- Enable component 'PluginTGeo' in 'Plugins/TGeo'
    -- Enable component 'PluginDD4hep' in 'Plugins/DD4hep'
    -- Ignore component 'Fatras' in 'Fatras'
    -- Ignore component 'Alignment' in 'Alignment'
    -- Ignore subdirectory 'Examples'
    -- Ignore subdirectory 'docs'
    -- Configuring done
    ```
    
3. Full log is here: [2023-02-01_acts-cmake-build-log.txt](https://github.com/acts-project/acts/files/10559406/2023-02-01_acts-cmake-build-log.txt)

4. Acts successfully compiled and installed

## What goes wrong

Our reconstruction framework using Acts have this in cmake:

```cmake
find_package(Acts REQUIRED COMPONENTS Core PluginIdentification PluginTGeo PluginDD4hep)
```

it gives an error: 

```
-- loading components:
--   Core
--   PluginActSVG
--   PluginIdentification
--   PluginJson
--   PluginTGeo
--   PluginDD4hep
CMake Error at CMakeLists.txt:77 (find_package):
  Found package configuration file:

    /home/romanov/eic/soft/acts/acts-v21.1.1/lib/cmake/Acts/ActsConfig.cmake

  but it set Acts_FOUND to FALSE so package "Acts" is considered to be NOT
  FOUND.  Reason given by package:

  The following imported targets are referenced, but are missing:
  actsvg::core actsvg::meta
```

(A bit confusing that it prints loading PluginActSVG when not asked but OK probably, right?)

## What I have tried

1. Build actsvg on my system, add it to cmake and LD paths
2. Used `-DACTS_USE_SYSTEM_ACTSVG=ON` and played with ACTSVG related flag combinations

## What goes as expected

Building with `-DACTS_BUILD_PLUGIN_ACTSVG=OFF` makes everything work

**Comment by paulgessinger** [comment]
@asalzburger did you look into this?

**Comment by paulgessinger** [comment]
@DraTeots is this still an issue?

**Comment by AJPfleger** [comment]
@asalzburger did you encounter this problem?

@DraTeots is this still an issue, with the current version?

**Comment by AJPfleger** [comment]
I had a look into this and I cannot reproduce the problem with the current version.

## Prerequisits
Now building with cpp20 instead of cp17
```
cmake -S . -B build -Wno-dev \
  -DCMAKE_INSTALL_PREFIX=install \
  -DCMAKE_CXX_STANDARD=20  \
  -DACTS_BUILD_PLUGIN_TGEO=ON  \
  -DACTS_BUILD_PLUGIN_DD4HEP=ON  \
  -DACTS_BUILD_PLUGIN_JSON=ON  \
  -DACTS_BUILD_PLUGIN_ACTSVG=ON 

cmake --build build -- -j64
cmake --build build --target install
```

## Try to reproduce bug

I created a minimal `CMakeLists.txt`:
```
cmake_minimum_required(VERSION 3.10)
project(ActsTest)

find_package(Acts REQUIRED COMPONENTS Core PluginTGeo PluginDD4hep)
```
without `PluginIdentification`, since it was removed in `v35.0.0` with https://github.com/acts-project/acts/pull/3177 .

I built the test-project with:
```
cmake -S . -B build \
  -Wno-dev \
  -DCMAKE_PREFIX_PATH=/home/apfleger/acts-issue-1818/install

cmake --build build
```
which succeeded.

The relevant log:
```
-- found Acts version 9.9.9 commit 9a2b74def
-- Including DD4hepBuild.cmake
-- Input:
-- Resolved to: 3.9.18
-- DD4hep uses Geant4
-- Found XercesC: /usr/lib64/libxerces-c.so (found suitable version "3.2.5", minimum required is "3.2.5")
-- WARNING: Mismatch in Python version: 3.9.18 vs. 3.12.2
--          ABI compatibility should not be assumed!
-- |++> Using python executable:  /usr/bin/python3.9
-- Found XercesC: /usr/lib64/libxerces-c.so (found version "3.2.5")
-- loading components:
--   Core
--   PluginActSVG
--   PluginJson
--   PluginTGeo
--   PluginDD4hep
-- Configuring done (0.1s)
-- Generating done (0.0s)
-- Build files have been written to: /home/apfleger/acts-issue-1818/acts_test/build
```
